bgstally/ui.py

```diff
@@ -290,3 +299,10 @@ def _confirm_force_tick(self):
         """
         answer = askyesno(title="Confirm FORCE a New Tick", message="This will move your current activity into the previous tick, and clear activity for the current tick.\n\nWARNING: It is not usually necessary to force a tick. Only do this if you know FOR CERTAIN there has been a tick but BGS-Tally is not showing it.\n\nAre you sure that you want to do this?", default="no")
         if answer: self.bgstally.new_tick(True, UpdateUIPolicy.IMMEDIATE)
+
+    def _overlay_options_state(self):
```

Comment:
Sorry, only just had time to come back to finish this review. Final comment - it doesn't look like this function is needed any longer?

Reply:
Good catch.. I wanted to avoid duplicating `state="disabled" if self.bgstally.overlay.edmcoverlay == None else "enabled",` knowing you want to add more checkboxes. Somewhere along the line I reverted back to having it duplicated. Let me fix that up so the method is actually being used.